### ISSUE TITLE
config file not loading when configured in application.properties

### ISSUE THREAD

**Issue by woutervantoledo** [akkinoc/logback-access-spring-boot-starter#110]
## Describe the bug

When you configure a config file in the applicaiton.properties of springboot the file can't load.
Setting used : logback.access.config=/opt/application/logback-access.xml
you get the error :

2021-12-24 09:38:34,868 WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext  - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Un
able to start web server; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'tomcatServletWebServerFactory' defined in class path resource [org/springframework/boot/autoconfigure/w
eb/servlet/ServletWebServerFactoryConfiguration$EmbeddedTomcat.class]: Initialization of bean failed; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'logbackAccessTomca
tWebServerFactoryCustomizer' defined in class path resource [dev/akkinoc/spring/boot/logback/access/tomcat/LogbackAccessTomcatConfiguration.class]: Unsatisfied dependency expressed through method 'logbackAccessTomcatWebServerFactoryCusto
mizer' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'logbackAccessContext' defined in class path resource [dev/akkinoc/spring/boot/logback/access/LogbackAccessAut
oConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [dev.akkinoc.spring.boot.logback.access.LogbackAccessContext]: Factory m
ethod 'logbackAccessContext' threw exception; nested exception is java.io.FileNotFoundException: class path resource [opt/application/logback-access.xml] cannot be resolved to URL because it does not exist
2021-12-24 09:38:34,918 INFO  o.s.b.a.l.ConditionEvaluationReportLoggingListener  

## To Reproduce

Create a springboot application with logback-access.xml on the file path.
create a application.properties with logback.access.config=/opt/application/logback-access.xml

## Expected behavior
File load also from local and not only from the classpath

### Environment
* **Version of this library used**:
* had this issue with all the 3.x versions of this library. The 2.x versions work correct.
* **Version of Java used**:
* Java 11

**Comment by akkinoc** [comment]
Thank you for the report.
Would you please try the `file:` prefix?

```
logback.access.config=file:///opt/application/logback-access.xml
```

**Comment by woutervantoledo** [comment]
That works like a charm.
Can you update the readme with this info for the next person running into this issue?

Thanks for the quick advice.

**Comment by akkinoc** [comment]
Thank you for trying.
I would like to add it to the README.
I have reopened this issue for my memorandum.

**Comment by akkinoc** [comment]
I added a note to the README about setting by "file:" scheme URI.
I also fixed this issue as a bug to make it behave the same as the Spring Boot "logging.config" property.
I have released it in v3.2.0.

Thanks.
